--- server/methods/deleteUser.js.new	2018-01-22 22:19:33.908625602 +0000
+++ server/methods/deleteUser.js 2018-01-22 22:27:02.646006989 +0000
@@ -25,12 +25,12 @@
 
 		const userIsAdmin = user.roles.indexOf('admin') > -1;
 
-		if (adminCount === 1 && userIsAdmin) {
-			throw new Meteor.Error('error-action-not-allowed', 'Leaving the app without admins is not allowed', {
-				method: 'deleteUser',
-				action: 'Remove_last_admin'
-			});
-		}
+		//if (adminCount === 1 && userIsAdmin) {
+		//	throw new Meteor.Error('error-action-not-allowed', 'Leaving the app without admins is not allowed', {
+		//		method: 'deleteUser',
+		//		action: 'Remove_last_admin'
+		//	});
+		//}
 
 		RocketChat.deleteUser(userId);
 
