--- packages/rocketchat-authorization/server/methods/removeUserFromRole.js	2018-01-22 22:19:33.820631883 +0000
+++ packages/rocketchat-authorization/server/methods/removeUserFromRole.js.new	2018-01-22 22:30:51.685111375 +0000
@@ -39,12 +39,12 @@
 			}).count();
 
 			const userIsAdmin = user.roles.indexOf('admin') > -1;
-			if (adminCount === 1 && userIsAdmin) {
-				throw new Meteor.Error('error-action-not-allowed', 'Leaving the app without admins is not allowed', {
-					method: 'removeUserFromRole',
-					action: 'Remove_last_admin'
-				});
-			}
+			//if (adminCount === 1 && userIsAdmin) {
+			//	throw new Meteor.Error('error-action-not-allowed', 'Leaving the app without admins is not allowed', {
+			//		method: 'removeUserFromRole',
+			//		action: 'Remove_last_admin'
+			//	});
+			//}
 		}
 
 		const remove = RocketChat.models.Roles.removeUserRoles(user._id, roleName, scope);
