--- programs/server/packages/rocketchat_authorization.js.orig	2018-01-23 21:15:44.118313055 +0000
+++ programs/server/packages/rocketchat_authorization.js	2018-01-23 21:16:45.268928353 +0000
@@ -798,12 +798,12 @@
 			}).count();
 			const userIsAdmin = user.roles.indexOf('admin') > -1;

-			if (adminCount === 1 && userIsAdmin) {
-				throw new Meteor.Error('error-action-not-allowed', 'Leaving the app without admins is not allowed', {
-					method: 'removeUserFromRole',
-					action: 'Remove_last_admin'
-				});
-			}
+			//if (adminCount === 1 && userIsAdmin) {
+			//	throw new Meteor.Error('error-action-not-allowed', 'Leaving the app without admins is not allowed', {
+			//		method: 'removeUserFromRole',
+			//		action: 'Remove_last_admin'
+			//	});
+			//}
 		}

 		const remove = RocketChat.models.Roles.removeUserRoles(user._id, roleName, scope);
